### PR TITLE
AppDomain needs protocol

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Please see [internal task](https://office.visualstudio.com/OC/_workitems/edit/2725287) for more details.

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[79b2bba](https://github.com/OfficeDev/office-js-docs-pr/pull/613/commits/79b2bba6e6ab681090c7dee7d51994c91d7aaa5d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/appdomain.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-protocol/docs/reference/manifest/appdomain.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/appdomains.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-protocol/docs/reference/manifest/appdomains.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812122159148981-613%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ebazPqpOTE7Mnahg7s3RSe6h%2FnN3sC7Klas%2BTDbNjoI%3D&st=2018-12-12T21%3A55%3A21Z&se=2019-01-12T22%3A00%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/appdomain.md [hunk 1]
```diff
@@ -7,14 +7,16 @@ Specifies an additional domain that will be used to load pages in the add-in win
 ## Syntax
 
 ```XML
-<AppDomain>string </AppDomain>
+<AppDomain>string</AppDomain>
 ```
 
+> [!IMPORTANT]
+> The **AppDomain** must include the protocol e.g., `<AppDomain>https://myappdomain"<AppDomain>`.
```
Comment (kbrandl) [inline docs/reference/manifest/appdomain.md]: Suggest rephrasing to -> The value of the **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain<AppDomain>`).

- puts "e.g.,..." in parenthesis
- removes the erroneous quotation mark from the end of the specified domain

File: docs/reference/manifest/appdomain.md [hunk 2]
```diff
@@ -7,14 +7,16 @@ Specifies an additional domain that will be used to load pages in the add-in win
...
+
 ## Contained in
 
 [AppDomains](appdomains.md)
 
 ## Remarks
 
-The  **AppDomains** and **AppDomain** elements are used to specify any additional domains other than the one specified in the SourceLocation element. For more information, see Office Add-ins XML manifest.
-
+The  **AppDomains** and **AppDomain** elements are used to specify any additional domains other than the one specified in the [SourceLocation element](sourcelocation.md). For more information, see [Office Add-ins XML manifest](/office/dev/add-ins/develop/add-in-manifests).
```
Comment (kbrandl) [inline docs/reference/manifest/appdomain.md]: Suggest rephrasing the start of this sentence -> "**AppDomain** elements should be used to specify..."

File: docs/reference/manifest/appdomains.md [hunk 3]
```diff
@@ -13,6 +13,9 @@ Lists any domains in addition to the domain specified in the SourceLocation elem
 </AppDomains>
 ```
 
+> [!IMPORTANT]
+> Each **AppDomain** must include the protocol e.g., `<AppDomain>https://myappdomain"<AppDomain>`.
```
Comment (kbrandl) [inline docs/reference/manifest/appdomains.md]: Suggest rephrasing to -> The value of each **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain<AppDomain>`).

- puts "e.g.,..." in parenthesis
- removes the erroneous quotation mark from the end of the specified domain

**Comment by o365devx** [comment]
OPS Build status updates of commit _[84c16df](https://github.com/OfficeDev/office-js-docs-pr/pull/613/commits/84c16df5b92e61222ddeac75e0829f7fddea3415)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/appdomain.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-protocol/docs/reference/manifest/appdomain.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/appdomains.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-protocol/docs/reference/manifest/appdomains.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812122356079347-613%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=W0f5PlZ2J3c6nj6WPbOIjyjT3%2Bj778cLQUkdUHyquDo%3D&st=2018-12-12T23%3A52%3A13Z&se=2019-01-12T23%3A57%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**